client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,98 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
+    class="c-support space-inset-m color--black bg-color--blue-light-3">
+    <h2 class="font-normal color--black">
+      {{ Translator.trans('support.heading') }}
+    </h2>
+    <p class="u-mt-0_75">
+      {{ Translator.trans('support.introduction') }}
+    </p>
+    <p>
+      {{ Translator.trans('support.contact.advice') }}
+    </p>
+    <h3 class="u-mt-0_75">
+      {{ Translator.trans('support.content') }}
+    </h3>
+    <ul
+      class="u-mb-0_75"
+      :class="contactLength === 1 ? '' : 'grid lg:grid-cols-3 gap-3'">
+      <li
+        v-for="contact in contacts"
+        :key="contact.id"
+        class="space-inset-m c-support__card color--black bg-color--white"
+        :class="contactLength === 1 ? 'lg:w-8/12' : ''">
+        <dp-faq-support-card
+          :title="contact.attributes.title"
+          :email="contact.attributes.eMailAddress"
+          :phone-number="contact.attributes.phoneNumber"
+          :reachability="contact.attributes.text" />
+      </li>
+    </ul>
+    <h3>
+      {{ Translator.trans('support.technical') }}
+    </h3>
+    <div class="lg:w-8/12 space-inset-m u-pv-0 c-support__card color--black bg-color--white">
+      <dp-faq-support-card />
```

Comment:
Why do You don't pass Props here and work with fallbacks instead? For my opinion using props here aswell would make the code in DpFaqSupportCard much cleaner

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/2139/commits/0efadeaeabe4f7a74c36fe22ec1634695b14cc22

Reply:
and here indentation https://github.com/demos-europe/demosplan-core/pull/2139/commits/bc342366e346fd32a63de5c66ab751cad6ff290e